Wrappers/Python/cil/processors/CofR_image_sharpness.py

```diff
@@ -15,47 +15,94 @@
 
-from cil.framework import Processor, AcquisitionData, AcquisitionGeometry
-from cil.processors.Binner import Binner
+from cil.framework import Processor, AcquisitionData, DataOrder
 import matplotlib.pyplot as plt
 import scipy
 import numpy as np
 import inspect
 import logging
 import math
+import importlib
 
 logger = logging.getLogger(__name__)
 
 class CofR_image_sharpness(Processor):
 
-    r'''CofR_image_sharpness processor maximises the sharpness of a reconstructed slice.
+    """
+    This creates a CentreOfRotationCorrector processor which will find the centre by maximising the sharpness of a reconstructed slice.
 
-    The centre-of-rotation offset is fitted using a reconstruction of edge-enhanced data. Auto-correlation is used to assess sharpness of the reconstructed slice. 
+    Can be used on single slice parallel-beam, and centre slice cone beam geometry. For use only with datasets that can be reconstructed with FBP/FDK.
 
-    For use on data-sets that can be reconstructed with FBP.
+    Parameters
+    ----------
 
-    :param slice_index: An integer defining the vertical slice to run the algorithm on.
-    :type slice_index: int, str='centre', optional
-    :param FBP: A CIL FBP class imported from cil.plugins.tigre or cil.plugins.astra  
-    :type FBP: class
-    :param tolerance: The tolerance of the fit in pixels, the default is 1/200 of a pixel. Note this is a stopping critera, not a statement of accuracy of the algorithm.
-    :type tolerance: float, default = 0.001    
-    :param search_range: The range in pixels to search either side of the panel centre. If `None` the width of the panel/4 is used. 
-    :type search_range: int
-    :param initial_binning: The size of the bins for the initial grid. If `None` will bin the image to a step corresponding to <128 pixels. Note the fine search will be on unbinned data.
-    :type initial_binning: int
-    :return: returns an AcquisitionData object with an updated AcquisitionGeometry
-    :rtype: AcquisitionData
-    '''
+    slice_index : int, str='centre', default='centre'
+        An integer defining the vertical slice to run the algorithm on.
 
-    def __init__(self, slice_index='centre', FBP=None, tolerance=0.005, search_range=None, initial_binning=None):
+    backend : str, default='astra'
+        The backend to use for the reconstruction, 'astra' or 'tigre'
+
+    tolerance : float, default=1./200.
+        The tolerance of the fit in pixels, the default is 1/200 of a pixel. This is a stopping criteria, not a statement of accuracy of the algorithm.
+
+    search_range : int, default 0.25*pixels_num_h
+        The range in pixels to search either side of the panel centre. If `None` a quarter of the width of the panel is used.  
+
+    initial_binning : int
+        The size of the bins for the initial grid. If `None` will bin the image to a step corresponding to <128 pixels. Note the fine search will be on unbinned data.
+
+
+    Kwargs
+    ------
```

Comment:
I guess this class doesn't have a page in the docs? But if it did, the kwargs section wouldn't render properly (see above)

Reply:
fixed!